src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -98,26 +106,38 @@ function makeColumnData(title: string, dataIndex: string, type: ColumnType, rend
 
 const AGG_COLUMN_TYPE = 'real';
 
-const mapColumn =
-  (mode: DisplayMode, bucketColumns: BucketColumn[]) => (column: AnonymizedResultColumn, columnIdx: number) => {
-    if (column.type === 'aggregate') {
-      switch (mode) {
-        case 'anonymized':
-          return [makeColumnData(column.name, columnIdx + '_anon', AGG_COLUMN_TYPE, renderValue)];
-        case 'combined':
-          return [
-            makeColumnData(column.name + ' (anonymized)', columnIdx + '_anon', AGG_COLUMN_TYPE, renderLowCountValue),
-            makeColumnData(column.name + ' (original)', columnIdx + '_real', AGG_COLUMN_TYPE, renderValue),
-            makeColumnData('Distortion', columnIdx + '_diff', 'real', renderRelativeNoiseValue),
```

Comment:
If we want some type safety here, we could use [template literals](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html):

```typescript
type ValueKey = number
type AggregatorKey = `${number}_anon` | `${number}_real` | `${number}_diff`
type Key = ValueKey | AggregatorKey
```

Reply:
The only hard coded value should be `lowCount` as it has special meaning for our UI. Everything else should be rendered automatically.